### ISSUE TITLE
Test fixtures for data files should have some kind of helpful file extension

### ISSUE THREAD

**Issue by garydgregory** [google/oss-fuzz#10362]
As a "user" of the issue https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=53669&q=label%3AProj-apache-commons-imaging I was confounded by the "Reproducer Testcase" file https://oss-fuzz.com/download?testcase_id=5965016805539840 which I mistakenly took to be a zip file instead of a TIFF file. 

I wish the file had a more helpful extension. If none can be computed by the tool, then I'd suggest using ".bin" to indicate this is an unknown binary file.

**Comment by jonathanmetzman** [comment]
Fair point. We're not going to get into guessing file extensions (I'm not sure how the extension is useful if the only reason to use the file is to reproduce the crash, like it's probably user hostile behavior if double clicking on a "PDF" file they downloaded from us crashes their browser).
.bin may be a good idea to add, but then again not all the fuzzer produced testcases are binary (JS for example).
